app/views/foreman/job_templates/install_errata_by_search_query_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,24 @@
+<%#
+kind: job_template
+name: Install errata by search query - Katello Ansible Default
+job_category: Katello
+description_format: 'Install errata %{Errata search query}'
+feature: katello_errata_install_by_search
+provider_type: Ansible
+template_inputs:
+- name: Errata search query
+  description: Filter criteria for errata to be installed.
+  input_type: user
+  required: false
+%>
+
+<% advisory_ids = @host.advisory_ids(search: input("Errata search query")) -%>
+<% render_error(N_("No errata matching given search query")) if !input("Errata search query").blank? && advisory_ids.blank? -%>
+# RESOLVED_ERRATA_IDS=<%= advisory_ids.join(',') %>
+
+<% if @host.operatingsystem.family == 'Suse' -%>
+<%= render_template('Run Command - Ansible Default', :command => "zypper -n install -t patch #{advisory_ids.join(' ')}") %>
+<% else -%>
+<% advisories = advisory_ids.map { |e| "--advisory=#{e}" }.join(' ') -%>
+<%= render_template('Run Command - Ansible Default', :command => "yum -y update-minimal #{advisories}") %>
```

Comment:
```suggestion
<%= render_template('Run Command - Ansible Default', :command => "yum -y update-minimal #{advisories}") -%>
```

Reply:
Why would the hyphen be needed at the end?

Reply:
Just cosmetic.

Reply:
https://www.puppet.com/docs/puppet/5.5/lang_template_erb.html#tags

The template renders the same, so maybe not needed?

```
# RESOLVED_ERRATA_IDS=RHBA-2023:0341

---
- hosts: all
  tasks:
    - shell:
        cmd: |
          yum -y update-minimal --advisory=RHBA-2023:0341
      register: out
    - debug: var=out
```

vs

```
# RESOLVED_ERRATA_IDS=RHBA-2023:1491

---
- hosts: all
  tasks:
    - shell:
        cmd: |
          yum -y update-minimal --advisory=RHBA-2023:1491
      register: out
    - debug: var=out
```

Reply:
IIRC it was just one end with \n while other have one \n less